mm2src/coins/utxo/utxo_tests.rs

```diff
@@ -2741,8 +2779,12 @@ fn test_generate_tx_doge_fee() {
     let builder = UtxoTxBuilder::new(&doge)
         .add_available_inputs(unspents)
         .add_outputs(outputs);
-    let (_, data) = block_on(builder.build()).unwrap();
-    let expected_fee = 1000000;
+    let (input_signer, data) = block_on(builder.build()).unwrap();
+    let transaction = UtxoTx::from(input_signer);
+    let v_size = tx_size_in_v_bytes(&UtxoAddressFormat::Standard, &transaction) as u64;
+    assert!(v_size < KILO_BYTE as u64);
```

Comment:
We should do this check to the couple of lines below as well.

Also the second time we calculate the tx size, I think the old transaction is used and not the new one.

Reply:
yeah thanks for the catch..I didn't finish modifying the test yet.

done now